File: src/main/java/aurocosh/divinefavor/common/network/message/client/syncing/MessageSyncEvilEye.java
package aurocosh.divinefavor.common.network.message.client.syncing;

import aurocosh.divinefavor.DivineFavor;
import aurocosh.divinefavor.common.custom_data.player.PlayerData;
import aurocosh.divinefavor.common.custom_data.player.data.curse.evil_eye.EvilEyeData;
import aurocosh.divinefavor.common.network.base.WrappedClientMessage;
import net.minecraft.entity.player.EntityPlayer;
import net.minecraftforge.fml.relauncher.Side;
import net.minecraftforge.fml.relauncher.SideOnly;

public class MessageSyncEvilEye extends WrappedClientMessage {
    public int severity;

    public MessageSyncEvilEye() {
    }

    public MessageSyncEvilEye(int severity) {
        this.severity = severity;
    }

    @Override
    @SideOnly(Side.CLIENT)
    protected void handleSafe() {
        EntityPlayer player = DivineFavor.proxy.getClientPlayer();
        EvilEyeData evilEyeData = PlayerData.INSTANCE.get(player).getEvilEyeData();
        evilEyeData.setSeverity(severity);
    }
}

File: src/main/kotlin/aurocosh/divinefavor/common/network/message/client/syncing/MessageSyncEvilEye.kt
package aurocosh.divinefavor.common.network.message.client.syncing

import aurocosh.divinefavor.DivineFavor
import aurocosh.divinefavor.common.lib.extensions.divinePlayerData
import aurocosh.divinefavor.common.network.message.base.DivineClientMessage
import net.minecraftforge.fml.relauncher.Side
import net.minecraftforge.fml.relauncher.SideOnly

class MessageSyncEvilEye : DivineClientMessage {
    var severity: Int = 0

    constructor() {}

    constructor(severity: Int) {
        this.severity = severity
    }

    @SideOnly(Side.CLIENT)
    override fun handleSafe() {
        val player = DivineFavor.proxy.clientPlayer
        player.divinePlayerData.evilEyeData.severity = severity
    }
}
